spec/controllers/media_controller_spec.rb

```diff
@@ -68,17 +68,17 @@
 
   describe '#stream' do
     let(:streaming_base_url) { Settings.stream.url }
-    let(:token_string) { 'tokenstring' }
-    let(:streaming_url_query_str) { "token_string=#{token_string}" }
+    let(:token) { 'encrypted_token_value' }
```

Comment:
how about `encrypted_token` for this var name?  in other parts of the code (including tests below), `token` is a var name for a `StacksMediaToken` object and not its encrypted string representation.  seems like it'd be nice to be consistent.